beacon-chain/blockchain/forkchoice/service.go

```diff
@@ -206,9 +204,13 @@ func (s *Store) latestAttestingBalance(ctx context.Context, root []byte) (uint64
 	if lastJustifiedState == nil {
 		return 0, errors.Wrapf(err, "could not get justified state at epoch %d", s.JustifiedCheckpt().Epoch)
 	}
+	justfiedState, err := stateTrie.InitializeFromProto(lastJustifiedState)
```

Comment:
Why do we need this? Why can't last justified state just be type `*stateTrie.BeaconState`

Reply:
Resolved